### DISCUSSION TITLE
Should we add a new `*Arr Suite` category on the website?

### DISCUSSION THREAD

**Discussion by havardthom** [community-scripts/ProxmoxVE#273] [General]
Should we add a new `*Arr Suite` category to contain all *Arr scripts?
```
homarr
radarr
sonarr
readarr
whisparr
prowlarr
bazarr
lidarr
tdarr
notifiarr
autobrr
recyclarr (soon)
```

**Comment by oOStroudyOo** [comment]
Yes but only if you have the ability to install more than 1 app at a time on the container, otherwise it's no different to running individual scripts.

**Comment by MickLesk** [comment]
@havardthom The question I have is, is this meant to be an AIO Install.sh or a separate category on the website?

**Comment by mosaati** [comment]
Maybe if it is a separate script that bundles them all with a selector.

It would be not different from running them one by one for sure. However, it will definitely help some users who are just starting the journey.

My 2 cents.

**Comment by havardthom** [comment]
This is just for adding a website category, not for making a script. I'm not gonna make that at least, separate LXCs for each service is much better imo.

**Reply by mosaati** [reply]
Oh I didn't mean all in one container.

Separate containers with one install script.

**Reply by havardthom** [reply]
Just run the separate commands? We can not know all different combination of services people want

**Reply by mosaati** [reply]
I agree. This is why I mentioned a selector.

**Comment by havardthom** [comment]
This was added in https://github.com/community-scripts/ProxmoxVE/pull/370. Closing this
